lib/macho/load_commands.rb

```diff
@@ -148,7 +148,11 @@ module MachO
   # represented, and no actual data. Used when a more specific class
   # isn't available/implemented.
   class LoadCommand < MachOStructure
-    # @return [Fixnum] the offset in the file the command was created from
+    # @return [MachO::MachOView] the raw view associated with the load command
+    attr_reader :view
+
+    # @return [Fixnum] the load command's offset in the source file
+    # @deprecated use {#view} instead
     attr_reader :offset
```

Comment:
Instead of duplicating the data, create a method `offset` that transparently forwards the query to the view, e.g.:

```ruby
def offset
  view.offset
end
```

Or if you want to fully embrace Ruby's standard library, have a look at the `Forwardable` module.